htdocs/stripe/admin/stripe.php

```diff
@@ -197,12 +197,12 @@
 	print '<span class="titlefield fieldrequired">'.$langs->trans("STRIPE_TEST_WEBHOOK_KEY").'</span></td><td>';
 	if ($conf->global->MAIN_FEATURES_LEVEL >= 2) {
 	    print '<input class="minwidth300" type="text" name="STRIPE_TEST_WEBHOOK_ID" value="'.$conf->global->STRIPE_TEST_WEBHOOK_ID.'">';
-        print ' &nbsp; '.$langs->trans("Example").': we_xxxxxxxxxxxxxxxxxxxxxxxx<br>';
+      print ' &nbsp; '.$langs->trans("Example").': we_xxxxxxxxxxxxxxxxxxxxxxxx<br>';
 	}
 	print '<input class="minwidth300" type="text" name="STRIPE_TEST_WEBHOOK_KEY" value="'.$conf->global->STRIPE_TEST_WEBHOOK_KEY.'">';
 	print ' &nbsp; '.$langs->trans("Example").': whsec_xxxxxxxxxxxxxxxxxxxxxxxx';
-	$out = img_picto('', 'object_globe.png').' '.$langs->trans("ToOfferALinkForTestWebhook").' ';
-    $url = dol_buildpath('/public/stripe/ipn.php?test', 3);
+	  $out = img_picto('', 'object_globe.png').' '.$langs->trans("ToOfferALinkForTestWebhook").' ';
+    $url = dol_buildpath('/public/stripe/ipn.php?test', 2);
```

Comment:
We must use here 3, because the URL we want Stripe to use for IPN is a the url view when we are outside of local network.
If you use 2, the value that will be presented will be the local url, for example  http://127.0.0.1
If you don't have https when you use 3, it means your setup of dolibarr is not correct and there is a s missing into the conf.php file where the external url is defined.
So we should keep 3 here, but we can add a warning to tell user a s is missing into its conf.php file if he want to use Stripe in live mode.

Reply:
indeed, there was mistake in my conf.php file... so we need a warning to remind to set https for using Stripe